src/sth_configuration.js

```diff
@@ -1,14 +1,21 @@
-/* globals module, process */
+/* globals module, process, require */
 
 (function() {
   "use strict";
 
+  var path = require('path');
+
   var ENV = process.env;
 
   var dbUsername = ENV.DB_USERNAME || '';
   var dbPassword = ENV.DB_PASSWORD || '';
 
   module.exports = {
+    LOG_LEVEL: ENV.LOG_LEVEL || 'info',
+    LOG_TO_CONSOLE: ENV.LOG_TO_CONSOLE !== 'false',
```

Comment:
No, no, it is just the opposite, I mean if the environment variables LOG_TO_CONSOLE and LOG_TO_FILE are not equal to 'false' (this is, their values are not explicitly set to 'false') the logs will be sent to console and to the file set ;)